std_msgs/Bool need_walls
---
std_msgs/Float32 walls
